src/components/MastheadUser.js

```diff
@@ -84,9 +84,12 @@ export default class MastheadUser extends React.Component<MastheadUserDefaultPro
     }
     if (this.props.username && this.props.username.length > 0) {
       return (
-        <div className="attivio-globalmast-user attivio-globalmast-separator before">
-          {this.props.username}
-          {dropdown}
+        <div 
+          className="attivio-globalmast-user attivio-globalmast-separator before"
+          data-test="Attivio-LoggedIn-User"
```

Comment:
`Attivio-LoggedIn-User` doesn't seem to follow the naming convention. However, this `div` doesn't appear to be something the user would interact with. Instead, wouldn't the Dropdown itself need `data-test` attribute? You could remove this and hold off on addressing this component.